### DISCUSSION TITLE
局域网内限制ip每秒查询次数

### DISCUSSION THREAD

**Discussion by wwushi** [kkkgo/PaoPaoDNS#188] [问答和帮助] [answered]
有没有参数能限制局域网内每个ip查询的次数？不想套adh了。

**Comment by kkkgo** [comment, marked as answer]
限制查询次数似乎没什么应用场景，普通的配置就能满足家用甚至中小企业。
如果真的要做，在防火墙上做限制更为合理，效率和资源占用更好，比如使用iptables。
